### PR TITLE
US483 ramina

### PR DESCRIPTION
**Pull request by raminka07**
I added my User story, please review

### CODE REVIEW

**Comment by Sara-Kaji** [comment]
You did good Ramina. Just two suggestions. 
First you didn't put US483 for the StepDef file, 
second since the US has 1 AC, why you don't put all number (less than 0 and more than 99) in one example table? 
And for the last point, you didn't consider Driver User.

**Comment by Sara-Kaji** [comment]
One question, what is the reason you locate additional Header webElement?

File: src/test/java/com/navfort/runners/CukesRunner.java
Comment (Sara-Kaji) [inline src/test/java/com/navfort/runners/CukesRunner.java:1]: You don't need to push CukedRunner. This is for your usage.

File: src/test/resources/features/US483_ErrorCalendarRepeatBox.feature
Comment (Sara-Kaji) [inline src/test/resources/features/US483_ErrorCalendarRepeatBox.feature:1]: I think it's better to merge these two Scenario. Make one Example table with headers "numberLessThan0", and "numberMoreThan99".
Also add "driver userType" in the Example table.

File: src/test/java/com/navfort/runners/CukesRunner.java
Comment (raminka07) [reply src/test/java/com/navfort/runners/CukesRunner.java:1]: Thank you Sara for such a deep review. i can't review deep like that!
so for the header web element, there are some tricky parts so I decided to handle them with actions. and about the merging two features together, i couldn't figure how should i do it. it is really very long to explain. is it okay if i show you and explain and answer all your questions at the meeting tonight?

**Comment by raminka07** [comment]
one more thing. i don't understand where you saw my name in the feature file. and i do have everything named with "US_483". i checked, and for some reason it doesn't show in the github, but i actually have correct naming in Intellij

File: src/test/java/com/navfort/runners/CukesRunner.java
Comment (Sara-Kaji) [reply src/test/java/com/navfort/runners/CukesRunner.java:1]: Your the most welcome Ramina. Sure, we can talk about that in meeting.

**Comment by Sara-Kaji** [comment]
> one more thing. i don't understand where you saw my name in the feature file. and i do have everything named with "US_483". i checked, and for some reason it doesn't show in the github, but i actually have correct naming in Intellij

Ramina, I double checked and realized you named your local branch "US483 Ramina". As far as I remember Troy said we should name them like USnumber.

**Comment by raminka07** [comment]
So i fixed everything, commit and push one more time. if you could review again please Sara

File: src/test/resources/features/US483_ErrorCalendarRepeatBox.feature
Comment (Sara-Kaji) [reply src/test/resources/features/US483_ErrorCalendarRepeatBox.feature:1]: All good Ramina.

File: src/test/java/com/navfort/step_definitions/US483_ErrorCalendarRepeatBox_StepDefs.java
Comment (Sara-Kaji) [inline src/test/java/com/navfort/step_definitions/US483_ErrorCalendarRepeatBox_StepDefs.java:1]: All good here as well.

**Review by Sara-Kaji — APPROVED** [review]
All good!

**Pull request merged**